carma_cooperative_perception/include/carma_cooperative_perception/sdsm_to_detection_list_component.hpp

```diff
@@ -47,7 +47,11 @@ class SdsmToDetectionListNode : public carma_ros2_utils::CarmaLifecycleNode
 
   auto sdsm_msg_callback(const input_msg_type & msg) const -> void
   {
-    publisher_->publish(to_detection_list_msg(msg, georeference_));
+    try {
+      publisher_->publish(to_detection_list_msg(msg, georeference_));
+    } catch (const std::runtime_error & e) {
+      RCLCPP_ERROR_STREAM(get_logger(), "Not converting SDSM to detection list: " << e.what());
```

Comment:
Low priority. Seems like bad logging language. Maybe failed to convert SDSM to detection list

Reply:
Updated